include/boost/process/v2/detail/impl/process_handle_windows.ipp

```diff
@@ -113,6 +123,33 @@ void check_running_(HANDLE handle, error_code & ec, DWORD & exit_status)
         ec = detail::get_last_error();
 }
 
+#if !defined(BOOST_PROCESS_V2_DISABLE_UNDOCUMENTED_API)
+void suspend_(HANDLE handle, error_code & ec)
+{
+  auto nt_err = NtSuspendProcess(handle);
+  if (nt_err > 0xC0000000)
+    ec = detail::get_last_error();
+}
+
+void resume_(HANDLE handle, error_code & ec)
+{
+
+  auto nt_err = NtResumeProcess(handle);
+  if (nt_err > 0xC0000000)
+    ec = detail::get_last_error();
```

Comment:
I'll need to check if that is actually correct.

Reply:
This doesnt seem correct according to NTSTATUS documentation but if NTSATUS is nonzero assume error. Very rarely will it be a function that succeeded properly with a nonzero value. At best it would be informational, which still means something that needs the developers attention is going on.

Reply:
I guess this can be marked as resolved as soon as I use your suggestion to convert NTSTATUS to a 'normal' win32 error code. I'll let you know once I've done this.